content/en/docs/setup/independent/high-availability.md

```diff
@@ -36,9 +36,6 @@ For both methods you need this infrastructure:
 - Three machines that meet [kubeadm's minimum
   requirements](/docs/setup/independent/install-kubeadm/#before-you-begin) for
   the masters
```

Comment:
Agree with @Bradamant3 . Maybe we can remove the below lines and change the above lines as:
... for the masters and three other machines for the workers.

Reply:
I could't agree more...  Just merge them with  only one hyperlink please